PopPUNK/sparse_mst.py

```diff
@@ -12,11 +12,13 @@
 import pandas as pd
 from scipy import sparse
 
-# GPU support
+# Load GPU libraries
 try:
+    import cupyx
     import cugraph
     import cudf
     import cupy as cp
+    from numba import cuda
```

Comment:
Is this used?

Reply:
It's in a few other files too, but I can't see it being used (but might just be missing it)